scripts/extract_documents.py

```diff
@@ -0,0 +1,66 @@
+""" A script to extract raw and rendered versions of all documents in the db
+
+This script extracts documents from the db into a testdocs/ directory.
+It is intended for creating a test corpus and verifying that changes
+to kuma and/or kumascript do not cause rendering changes to the documents.
+
+In order to make it work it may be necessary to modify the KUMASCRIPT_TIMEOUT
+setting in kuma/settings/common.py to be at least 60 seconds
+"""
+
+import json
+import os
+import os.path
+import sys
+from datetime import datetime
+
+# Adjust the path so the imports below all work.
+sys.path[0] = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+import django
+from django.conf import settings
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'kuma.settings.testing')
+django.setup()
```

Comment:
This suggests this should be a management command rather than a stand-alone script.

Reply:
It was useful to me while developing and testing this change. But I'm not sure whether it is generally useful enough to warrant further work. If you don't want to land this as-is, I'd be inclined to just remove it rather than invest more time in it.